pkg/skaffold/initializer/init.go

```diff
@@ -74,11 +84,17 @@ func DoInit(ctx context.Context, out io.Writer, c config.Config) error {
 	}
 
 	if !c.Force {
-		if done, err := prompt.WriteSkaffoldConfig(out, pipeline, c.Opts.ConfigurationFile); done {
+		if done, err := prompt.WriteSkaffoldConfig(out, pipeline, generatedManifests, c.Opts.ConfigurationFile); done {
 			return err
 		}
 	}
 
+	for path, manifest := range generatedManifests {
+		if err := ioutil.WriteFile(path, manifest, 0644); err != nil {
```

Comment:
shd we also add the files generated at L102 at the summary along with 
```
fmt.Fprintf(out, "Configuration %s was written\n", c.Opts.ConfigurationFile)
```

Reply:
sure, added